### ISSUE TITLE
timezone imported from wrong module?

### ISSUE THREAD

**Issue by Digitante** [Patreon/patreon-python#3]
Does not import properly when tested in Python 2.7.8, failing on this:

from datetime import timezone

There is no "timezone" object to import from the "datetime" module. There is one in the "time" module -- perhaps this is what was intended?  I have not yet found out whether this is really the same object, but so far, the module works with this line altered to:

from time import timezone

**Comment by 21echoes** [comment]
The code was written for python 3+, where `timezone` is a module that provides timezone definitions. it is used to convert all cursor dates to UTC before sending them to the Patreon server. I'll see if we can backport this library to python 2 this week :D

**Comment by Digitante** [comment]
Okay. But it doesn't import in Python 3.4.1, either. I got a different bug
with that:

ImportError: No module named 'requests'

Not sure when the "requests" module would have been added (although it
apparently exists in 2.7).

Note that after correcting the import to load "timezone" from the "time"
module, the code does work. I suppose it might be getting the times wrong
or something, but it hasn't given me a traceback.

Probably a good idea to note what versions of Python you're testing against.


On Mon, Jun 20, 2016 at 1:15 PM, David Kettler <notifications@github.com>
wrote:

> The code was written for python 3+, where timezone is a module that
> provides timezone definitions. it is used to convert all cursor dates to
> UTC before sending them to the Patreon server. I'll see if we can backport
> this library to python 2 this week :D
>
>
--
Terry Hancock
Producer/Director - "Lunatics!" Project - http://lunatics.tv
